pkg/controller/node/node_controller.go

```diff
@@ -593,7 +593,7 @@ func (nc *Controller) monitorNodeStatus() error {
 		}
 		instance, isSupported := nc.cloud.Instances()
 		if isSupported {
-			instance.NodeRegistered(added[i])
+			go instance.NodeRegistered(added[i])
```

Comment:
I have floated review request on kubernetes directly

Reply:
Removed this from this PR